sunpy/net/dataretriever/sources/adapt.py

```diff
@@ -0,0 +1,74 @@
+# # Author: Gilly <gilly@swri.org>
+# import sunpy.net
+
+from sunpy.net import attrs
+from sunpy.net.fido_factory import Fido
+from sunpy.net.dataretriever import GenericClient
+from sunpy.net.dataretriever.attrs.adapt import *
+import sunpy.net.dataretriever.attrs as atrs
+from sunpy.net.attr import SimpleAttr
+
+
+__all__ = ['ADAPTClient']
+
+
+class ADAPTClient(GenericClient):
```

Comment:
We will need a documentation string that follows the format of the others. You can just copy and paste another one from another client and just find and replace some names.